packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/updateJobs.st

```diff
@@ -6,10 +6,14 @@ updateJobs
 	id := self parser parseIdOfWorkflowAt: self selectedHistory ofWorkflowsOfRepository: self data repositoryJson.
 	client := ATDDGitHubActionsClient new.
 	jobs := self parser parseJobsOf: ( client getJsonForJobsForRepository: (self data repository , '/') forWorkflowRunId: id ).
-
-	self jobs: (jobs collect: [ :job |
-		Text new append: '#' , ( self parser primitiveParseNameOf: job );
-		addAttribute: ( TextColor color: (self colorForStatus: (self parser primitiveParseLatestWorkflowRunConclusion: job))) ] );
+	
+	self jobs: (jobs collect: [ :job | ATDDGitHubActionWorklowJob newForId: (parser primitiveParseIdOf: job) 
+		andName: (parser primitiveParseNameOf: job) 
+		andConclusion: (parser primitiveParseLatestWorkflowRunConclusion: job) ] ).
+	
+	self jobNames: (self jobs collect: [ :job |
+		Text new append: '#' , job name;
+		addAttribute: ( TextColor color: (self colorForStatus: job conclusion)) ] );
```

Comment:
Why do we need `jobNames` when we already have `jobs` which contain the exact same names? Feels like storing data twice (redundant)

Reply:
hmm not exactly, we need the names for the model, so that it gets updated whenever they change, but we also need them as an object